modules/circl/1.3.8/source.json

```diff
@@ -0,0 +1,14 @@
+{
+    "url": "https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.3.8.zip",
+    "integrity": "sha256-gCOgRORw9SLiykG62YsYrW0TTWiBABsRIK0ukWbNpuY=",
+    "patches": {
+        "0001-build-add-bzlmod-files.patch": "sha256-VJjrWhKx38f80vGBjwMgXQHiTaidlX1l59F8T/pi5dE=",
```

Comment:
You can reuse old patches via `../../<other_version>/patches/patch`, which may help reduce the diff.